<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 196, 255)</string>
  </property>
  <widget class="QPushButton" name="spring_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 25;
border-width: 3px;
border-style: outset;
border-color: rgb(0, 89, 168);
background-color: rgb(79, 226, 255)</string>
   </property>
   <property name="text">
    <string>Весна</string>
   </property>
  </widget>
  <widget class="QPushButton" name="summer_button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 25;
border-width: 3px;
border-style: outset;
border-color: rgb(0, 89, 168);
background-color: rgb(79, 226, 255)</string>
   </property>
   <property name="text">
    <string>Лето</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fall_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 25;
border-width: 3px;
border-style: outset;
border-color: rgb(0, 89, 168);
background-color: rgb(79, 226, 255)</string>
   </property>
   <property name="text">
    <string>Осень</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Выберите время года:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
